Assets/Scripts/UI/InGame/HUD/CustomizableHUDElement.cs

```diff
@@ -0,0 +1,243 @@
+﻿using UnityEngine.EventSystems;
+using UnityEngine;
+using Assets.Scripts.UI.InGame;
+
+/* Any HUD element that is customizable should have this script attached */
```

Comment:
Use `///sumary` above the class with that information, as this comment is fairly easy to miss and thus useless